modules/data.atmosphere/R/solar_angle.R

```diff
@@ -0,0 +1,36 @@
+#' Calculate solar angle
+#'
+#' @author Alexey Shiklomanov
+#' @param doy Day of year
+#' @param lat Latitude
+#' @param lon Longitude
+#' @param dt Timestep
```

Comment:
could be better documented. 
* What is the return value? 
* cite a canonical source of the equations used
* add description

Should the name be cos_solar_zenith_angle to be more explicit?